here-naksha-lib-base/src/commonTest/kotlin/naksha/base/PlatformTest.kt

```diff
@@ -5,14 +5,16 @@ import kotlin.test.*
 class PlatformTest {
     // TODO: Fix the commented out type casting
     @Test
-    fun fromJSON() {
+    fun testFromJSON() {
+        Platform.parseDataUrl = true
```

Comment:
Please change the fromJSON signature and add an additional parameter, so make it:
```kotlin
fun fromJSON(json: String, options: JsonOptions = JsonOptions.DEFAULT)
class JsonOptions(val parseDataUrl: Boolean = false) {
  companion object {
    @JvmStatic
    @JsStatic
    val DEFAULT = JsonOptions()
  }
}
```